risc0/zkvm/Cargo.toml

```diff
@@ -80,6 +80,7 @@ num-derive = "0.4"
 num-traits = { version = "0.2", default-features = false, optional = true }
 prost = { version = "0.12", optional = true }
 sha2 = { version = "0.10", optional = true }
+tempfile = "3"
```

Comment:
I think this will break the `no_std` test for the verifier. I suggest setting this as optional = true and then enabling only for the prove feature.

Reply:
good catch!